The system cannot find the file specified.
The system cannot find the file specified.
Traceback (most recent call last):
  File "c:\python32x86\lib\site-packages\win32\lib\win32verstamp.py", line 120, in stamp
    bits = [int(i) for i in ver.split(".")]
  File "c:\python32x86\lib\site-packages\win32\lib\win32verstamp.py", line 120, in <listcomp>
    bits = [int(i) for i in ver.split(".")]
ValueError: invalid literal for int() with base 10: '2012-10-21'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "setup.py", line 271, in <module>
    executables = cx_FreezeExecutables,
  File "c:\python32x86\lib\site-packages\cx_Freeze\dist.py", line 359, in setup
    distutils.core.setup(**attrs)
  File "c:\python32x86\lib\distutils\core.py", line 149, in setup
    dist.run_commands()
  File "c:\python32x86\lib\distutils\dist.py", line 919, in run_commands
    self.run_command(cmd)
  File "c:\python32x86\lib\distutils\dist.py", line 938, in run_command
    cmd_obj.run()
  File "c:\python32x86\lib\site-packages\cx_Freeze\dist.py", line 231, in run
    freezer.Freeze()
  File "c:\python32x86\lib\site-packages\cx_Freeze\freezer.py", line 470, in Freeze
    self._FreezeExecutable(executable)
  File "c:\python32x86\lib\site-packages\cx_Freeze\freezer.py", line 160, in _FreezeExecutable
    self._AddVersionResource(exe.targetName)
  File "c:\python32x86\lib\site-packages\cx_Freeze\freezer.py", line 105, in _AddVersionResource
    stamp(fileName, versionInfo)
  File "c:\python32x86\lib\site-packages\win32\lib\win32verstamp.py", line 123, in stamp
    raise ValueError("--version must be a.b.c.d (all integers) - got %r" % ver)
ValueError: --version must be a.b.c.d (all integers) - got '2012-10-21.0.0.0'
Traceback (most recent call last):
  File "c:\python32\lib\site-packages\win32\lib\win32verstamp.py", line 120, in stamp
    bits = [int(i) for i in ver.split(".")]
  File "c:\python32\lib\site-packages\win32\lib\win32verstamp.py", line 120, in <listcomp>
    bits = [int(i) for i in ver.split(".")]
ValueError: invalid literal for int() with base 10: '2012-10-21'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "setup.py", line 271, in <module>
    executables = cx_FreezeExecutables,
  File "c:\python32\lib\site-packages\cx_Freeze\dist.py", line 359, in setup
    distutils.core.setup(**attrs)
  File "c:\python32\lib\distutils\core.py", line 149, in setup
    dist.run_commands()
  File "c:\python32\lib\distutils\dist.py", line 919, in run_commands
    self.run_command(cmd)
  File "c:\python32\lib\distutils\dist.py", line 938, in run_command
    cmd_obj.run()
  File "c:\python32\lib\site-packages\cx_Freeze\dist.py", line 231, in run
    freezer.Freeze()
  File "c:\python32\lib\site-packages\cx_Freeze\freezer.py", line 470, in Freeze
    self._FreezeExecutable(executable)
  File "c:\python32\lib\site-packages\cx_Freeze\freezer.py", line 160, in _FreezeExecutable
    self._AddVersionResource(exe.targetName)
  File "c:\python32\lib\site-packages\cx_Freeze\freezer.py", line 105, in _AddVersionResource
    stamp(fileName, versionInfo)
  File "c:\python32\lib\site-packages\win32\lib\win32verstamp.py", line 123, in stamp
    raise ValueError("--version must be a.b.c.d (all integers) - got %r" % ver)
ValueError: --version must be a.b.c.d (all integers) - got '2012-10-21.0.0.0'
